js/cards/parsecards.py

```diff
@@ -72,6 +73,12 @@ def getLegalities(card, cards):
     elif cards[card]['colors'] == ['Red']:    ocards[ocard]['c'] = 'D'
     elif cards[card]['colors'] == ['Green']:  ocards[ocard]['c'] = 'E'
 
+    if   'Land'     in cards[card]['types']: ocards[ocard]['t'] = '1'
```

Comment:
Can we get an extra space between the test case (`'Land'`) and the `in` keyword so that the type lines up with the the color type lineup above?